### ISSUE TITLE
Metal noise in sample

### ISSUE THREAD

**Issue by trfnhle** [liusongxiang/efficient_tts#2]
Thanks for your great work. I have already heard some samples at [link](https://github.com/liusongxiang/efficient_tts/tree/main/egs/lj/current-gen-waves-130000steps). There are some metal noises at high pitch and some words are mispronounced. Does longer training overcome this problem? or is it caused by vocoder? I wonder how quality audio of efficient TTS + another vocoder(Melgan)?

**Comment by liusongxiang** [comment]
Thank you for your attention and the questions.
"Does longer training overcome this problem?"
I have uploaded new generated samples [https://github.com/liusongxiang/efficient_tts/tree/main/egs/lj/checkpoint-295000steps](url), which are better but still not good. Please take a look. I found the model overfits when trained for longer time, but the performance is increasing. I tried to add some dropout into the model to overcome the overfitting issue, but weirdly, the performance degrades although less overfitting. The simgas in Eq (14) and Eq (17) is not presented in the paper, and they are the additioinal important hyperparameters to tune.

"or is it caused by vocoder? I wonder how quality audio of efficient TTS + another vocoder(Melgan)?"
Currently I use the HifiGAN vocoder pretrained by https://github.com/jik876/hifi-gan, which has been tested having really good copy-synthesis performance. In the HifiGAN paper (https://arxiv.org/pdf/2010.05646v2.pdf), it is claimed that HifiGAN is better than MelGAN. So I will still use HifiGAN as the vocoder.

I am still tuning the model and hope to obtain the performance presented in the EfiicientTTS paper.

**Comment by dathudeptrai** [comment]
@liusongxiang so, even you are a second author, you still not know the real sigmas ur team used in the paper ? :D
